Makefile.am

```diff
@@ -15,6 +15,7 @@ nodist_noinst_DATA =
 nodist_noinst_SCRIPTS =
 
 TESTS = \
+	test/check-dist \
```

Comment:
This is not a static code test.  It requires you to do a full build first.

Also: `test/static-code` skips everything if you're not in a git repository, so this check is also skipped on tarball builds.

Ideally, I'd love to drop this test entirely and implement the check as part of the webpack build, but my brain is too weak for that level of hackery (unless I were to do it in `webpack-make`, which would be easy enough, I guess)...

Reply:
...or the Makefile

Reply:
If it needs to go anywhere else than test/static-code, it can just as well remain its own script. I was just wondering whether you did that just for some æsthetic reason, or if there is some technical requirement behind it? The commit message doesn't say, and it's not obvious why it needs to happen.

Reply:
Ya, basically just for the reason I wrote above.  Using this in combination with the git hooks which check out the code into the tmp directories and run tests on it sort of drives the point home: there will absolutely never be a `dist/` there.